tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) QuantCo 2023-2024
```

Comment:
I'd prefer if you add back `pythonpath` in the pytest options and remove this `__init__.py`.